.github/workflows/main.yml

```diff
@@ -94,3 +94,4 @@ jobs:
     uses: konveyor/ci/.github/workflows/global-ci.yml@main
     with:
       component_name: tackle2-hub
+      api_hub_tests_ref: ${{ github.head_ref }}
```

Comment:
I think we might want `github.ref_name` here instead, I think `head_ref` will reference just the PR branch while `ref_name` will reference the merged PR

Reply:
This workflow should affect only PRs. Once the change is merged the branch name (main and e.g. release-0.3) should be driven by already existing API_TESTS_REF.

Reply:
I think the difference is `head_ref` targets `/refs/pull/<PR #>/head` whereas `ref_name` targets `/refs/pull/<PR #>/merge`. Generally we want to test with merge so that we get the results of the tests _if that PR merged_. Can help catch issues that are introduced on the main branch after a PR is introduced. Let me know if I'm missing something or talking past you

Reply:
Ah, good point, I misuderstood it before, thanks! I will update.